modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/AlterTable.scala

```diff
@@ -90,3 +90,11 @@ case class AddColumn(
 case class DropColumn(columnName: String, mode: Option[DropMode]) extends Ddl {
   def toDdl = s"DROP COLUMN $columnName${mode.map(" " + _.toDdl).getOrElse("")}"
 }
+
+case class IncreaseMaxLength(
```

Comment:
I tried to call all classes against their DDL statements, e.g. `DROP COLUMN` = `DropColumn`, `RENAME COLUMN` = `RenameColumn` etc. I don't think it was a particularly good idea to mirror the whole language here, but think we can call this one `AlterType` for consistency.

Feel free to ignore though if you think we should stick to more concrete actions.